src/components/UserInfo/UserInfo.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+import axios from 'axios';
+import history from '../../history';
+import UserService from '../../Service/UserService.js';
+import style from './style.scss';
+
+export default class UserInfo extends Component {
+  state = {
+    newName: '',
+    snackMessage: '',
+    vertical: 'bottom',
+    horizontal: 'center',
+    openSnack: false
+  };
+
+  handleClose = () => {
+    this.setState({ open: false });
+  };
+
+  snack = message => {
+    this.setState({
+      snackMessage: message,
+      openSnack: true
+    });
+
+    setTimeout(
+      () =>
+        this.setState({
+          openSnack: false
+        }),
+      3000
+    );
+  };
+
+  changeInfo = () => {
+    const { newName } = this.state;
+    !newName
+      ? this.snack('Заполните поле')
```

Comment:
don't use ternary operator like this, it's really hard to read. 'if' would be much better in such cases